### DISCUSSION TITLE
New main branch

### DISCUSSION THREAD

**Discussion by jayofelony** [jayofelony/pwnagotchi#227] [Announcements]
As of now the main default branch will be `No AI`, the next release will have no more AI. This is a decision made after thorough testing. The AI software tends to break the custom WiFi firmware on a deep level, which destabilizes the WiFi. Which means that the Pwnagotchi will go blind more often, causing unwanted rebooting of the device.

In the next release, there will be a substitute for AI, a plugin called `Auto-Tune`. Made by [Sniffleupagus](https://github.com/Sniffleupagus/).

And yes, there is a new version (2.9.2) coming very very soon. I finally have a breakthrough in a new image building method. So stay tuned!! This means 32bit will also be having the latest kernel with bookworm Raspios.

**Comment by krishenriksen** [comment]
Can you include socat in 2.9.2 ?

socat:
  Installed: (none)
  Candidate: 1.7.4.4-2
  Version table:
     1.7.4.4-2 500
        500 http://deb.debian.org/debian bookworm/main arm64 Packages
        
        
socat:
  Installed: 1.7.4.1-3
  Candidate: 1.7.4.1-3
  Version table:
 *** 1.7.4.1-3 500
        500 http://raspbian.raspberrypi.org/raspbian bullseye/main armhf Packages
        
for pull request https://github.com/jayofelony/pwnagotchi/pull/233
<img width="561" alt="Screenshot 2024-11-30 at 01 35 47" src="https://github.com/user-attachments/assets/f22b3211-e36c-419e-97b9-f4214d69bc55">

**Reply by jayofelony** [reply]
Yep, on it.

**Reply by jayofelony** [reply]
If you are not, you can join our discord makes it easier to talk

https://discord.gg/CbqCeYF6

**Comment by benjamincarmel** [comment]
Will the next release be based off RPI OS Bookworm (November 19th 2024)?

**Reply by jayofelony** [reply]
Yes and no. I use pi-gen now to build images.
